Jenkinsfile.build

```diff
@@ -1,15 +1,18 @@
 // one job per arch (for now) that just builds "the top thing" (triggered by the meta-update job)
 properties([
-	disableConcurrentBuilds(),
```

Comment:
Oof, hmm, something to think about here:

https://github.com/docker-library/meta/blob/f595ec2375e11540c4e26352274b05247953d763/.github/workflows/build.yml#L28-L30

I wish Jenkins had something like GitHub's concurrency groups -- this prevents the same `buildId` from even _attempting_ to build twice at the same time, and with this PR the only thing that prevents that is that these are canonically triggered by the "trigger" job, which is fine for the normal case, but when things go wrong and we're running "build" by hand to debug, nothing will stop "trigger" from firing and potentially clobbering the thing we're testing. :thinking:

I think the closest thing Jenkins has is "Lockable Resources" (https://plugins.jenkins.io/lockable-resources/), but they're a really awful experience and IIRC nothing cleans them up, so we can't reasonably put an arbitrary number of those into the system. :sob:

Reply:
I think the ["Throttle Concurrent Builds" plugin](https://plugins.jenkins.io/throttle-concurrents/) can at least get things with the same parameters from running at the same time: https://www.gusi.me/2022/05/06/Disable-concurrent-builds-based-on-parameters.html. It will still be quasi-queued though.

Reply:
```groovy
	throttleJobProperty(
		limitOneJobWithMatchingParams: true,
		paramsToUseForLimit: 'identifier, buildId',
		throttleEnabled: true,
		throttleOption: 'project',
	),
```
Ok, I did a local test using the "Throttle Builds" plugin with this job config and found that it can stop concurrent jobs from adding to the Jenkins queue when specific parameters match already queued/running jobs.

![image](https://github.com/user-attachments/assets/ea5c2c54-c412-41d0-a135-63bfbf606144)

If the build job activation comes from something like the trigger job that uses `waitForStart :true`, then the trigger job will be stuck waiting to add it to the queue and not start anything else. This should be fine since trigger is the only one that should be starting them and so would only happen if we manually had started a build.